SwiftlyUserDefault/SwiftlyUserDefault/SwiftlyUserDefault.swift

```diff
@@ -12,31 +12,30 @@ public protocol SwiftlyUserDefaultable {
     var key: String { get }
 }
 
+@available (*, deprecated, message: "Use `UserDefaults` extension directly.")
```

Comment:
There's many ways of deprecating. I had to look it up and found this article by Paul Hudson: https://www.hackingwithswift.com/example-code/language/how-to-use-available-to-deprecate-old-apis